app/styles/_tables.less

```diff
@@ -197,3 +197,22 @@
     font-family: @font-family-monospace;
   }
 }
+
+// Secrets table
+// Set column widths so columns align across multiple tables per page
+// Allocate majority of table width to 'Name' and 'Type' columns, while 'Created' gets remainder
+.secrets-table tbody td,
+.secrets-table thead th {
+	&:nth-child(1) {
+		.word-break-all;
+		@media (min-width: 768px) { width: 40%;}
+		@media (min-width: 992px) { width: 42%;}
+		@media (min-width: 1200px) { width: 44%;}
+	}
+	&:nth-child(2) {
+		.word-break-all;
+		@media (min-width: 768px) { width: 40%;}
+		@media (min-width: 992px) { width: 42%;}
+		@media (min-width: 1200px) { width: 44%;}
+	}
+}
```

Comment:
I'd rather give the columns class names and set the styles based on class name. nth child can break if we ever change the table contents, and it won't be obvious editing the view.

```css
.name-col, .type-col {
  .word-break-all;
  @media (min-width: 768px) { width: 40%;}
  @media (min-width: 992px) { width: 42%;}
  @media (min-width: 1200px) { width: 44%;}
}
```